app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt

```diff
@@ -271,39 +271,41 @@ class SkillDetailsFragment : Fragment(), ISkillDetailsView {
         fiveStarSkillRatingBar = five_star_skill_rating_bar
         fiveStarSkillRatingScaleTextView = tv_five_star_skill_rating_scale
 
-        tvFiveStarSkillRatingBar.visibility = View.VISIBLE
-        fiveStarSkillRatingBar.visibility = View.VISIBLE
+        if (fiveStarSkillRatingBar != null) {
```

Comment:
Rating Bar should never be null

Check why it is instead of adding a null check